src/potions.ts

```diff
@@ -162,7 +162,8 @@ export class Potion {
       ? historical && historicalAge(this.potion) < 14
         ? historicalPrice(this.potion)
         : mallPrice(this.potion)
-      : retrievePrice(this.potion);
+      : retrievePrice(this.potion, itemAmount(this.potion) + 1) -
+          autosellPrice(this.potion) * itemAmount(this.potion);
```

Comment:
I would somehow slice out this logic somewhere and name it so that it makes a little more sense

Also, I think we do actively want to use availableAmount, unless we manually set all of the various autoSatisfy settings. Which, maybe we do!